src/Controllers/ArticleController.php

```diff
@@ -1,68 +1,91 @@
 <?php
 
 declare(strict_types=1);
-namespace App\Controllers;
 
+namespace App\Controllers;
 
-use App\Models\Main;
-use App\Models\Db;
-use App\Entities\ArticleEntity;
+use App\Services\ArticleService;
+use App\Controllers\TemplateController;
 
-class ArticleController extends Main
+class ArticleController extends ArticleService
 {
-    protected static string $table = 'posts';
+    private $prefix = '';
```

Comment:
```suggestion
    private string $prefix = '';
```

On est en POO et PHP 8 désormais